src/Game.cpp

```diff
@@ -88,7 +250,8 @@ int sub_401190() {
             sub_612EB0("movie", 0, &var4);
             glo_6C2E88.sub_53ACB0(var4);
             t = 15;
-        } else if (sub_612E10("skipintro") || glo_6C2C64 != 0) t = 11;
+        }
+        else if (sub_612E10("skipintro") || glo_6C2C64 != 0) t = 11;
```

Comment:
Why is this on a new line? I understand this as being a style issue, but I personally prefer it on the same line as the brace, as it is for the other if-else (and similar) blocks, even if this is one-lined.